packages/terra-data-grid/src/utils/WorklistDataGridUtils.js

```diff
@@ -2,7 +2,7 @@
  * The Object representation of the row selection column.
  */
 const ROW_SELECTION_COLUMN = {
-  id: 'WorklistDataGrid-rowSelectionColumn',
+  id: 'table-rowSelectionColumn',
```

Comment:
This will require a major version bump for the WorklistDataGrid.  We also need to update the appropriate examples and communicate the change to our consumers for Fusion Passthrough.